frostsnapp/native/src/coordinator.rs

```diff
@@ -108,25 +125,14 @@ impl FfiCoordinator {
                 }
             }
             // to give time for the other threads to get a lock
-            std::thread::sleep(Duration::from_millis(10));
+            std::thread::sleep(Duration::from_millis(100));
```

Comment:
Trying to reduce CPU on desktop?

Reply:
Yes. Every 100ms seems to be fast enough for ui now. Will take some work to investigate why so much CPU is being used.